### ISSUE TITLE
role="menuitem" does not support aria-expanded in ARIA 1.1, but the authoring practices requires this

### ISSUE THREAD

**Issue by katiehockman** [w3c/aria-practices#608]
"A parent menuitem has aria-expanded set to false when its child menu is not visible and set to true when the child menu is visible."
https://www.w3.org/TR/wai-aria-practices/#menu

This would mean that the element with role="menuitem" toggles aria-expanded depending on whether or not the child submenu is open. However, according to [ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/#menuitem), aria-expanded is not supported for role="menuitem".

There either needs to be a change to the ARIA 1.1 spec to include aria-expanded as a supported state, or a change to the authoring practices guide (and the associated example that follows this practice).

**Comment by mcking65** [comment]
the `menuitem` role supports `aria-expanded` in ARIA 1.2 with [ARIA commit 5bc57d4](https://github.com/w3c/aria/commit/5bc57d4899feacc5670e5de7e5d3eeb8f291f9cb) 
and this change is supported in the current version of the HTML validator.

We previously had notes about validation errors as documented in issue #447, but we have since removed them.
